src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -42,101 +42,104 @@
 from ...utils.str_utils import human_readable_size
 
 
-def _get_default_ophys_metadata() -> DeepDict:
-    """Fill default metadata for Device and ImagingPlane."""
-    metadata = get_default_nwbfile_metadata()
-
-    default_device = dict(name="Microscope")
-
-    default_optical_channel = dict(
-        name="OpticalChannel",
-        emission_lambda=np.nan,
-        description="An optical channel of the microscope.",
-    )
+def _get_default_ophys_metadata():
+    """
+    Returns fresh ophys default metadata dictionary.
 
-    default_imaging_plane = dict(
-        name="ImagingPlane",
-        description="The plane or volume being imaged by the microscope.",
-        excitation_lambda=np.nan,
-        indicator="unknown",
-        location="unknown",
-        device=default_device["name"],
-        optical_channel=[default_optical_channel],
-    )
+    Single source of truth for all ophys default metadata.
+    Structure matches the output of get_nwb_imaging_metadata() and get_nwb_segmentation_metadata().
+    Each call returns a new instance to prevent accidental mutation of global state.
```

Comment:
We should add tests for each of these two properties.

Reply:
I added a test, I also call the `_get_default_ophys_metadata` in the other tests to add more complex checks that is not being modified under the hood.